example/iam/config.yml

```diff
@@ -94,6 +101,12 @@ lf:
     protected-2:
       name: lf-protected-2
       type: protected
+    dashboard:
```

Comment:
Are `dashboard` and `dashboard-2` supposed to overwrite the default buckets?  The default bucket types are declared as `sandbox-dashboard` and `sit-dashboard` earlier.

Reply:
yeah I can remove them, I was just using those for my testing.